### PR TITLE
Add calibration face box

### PR DESCRIPTION
**Pull request by Scoobydoo181**
Capture the position of the user's face while they're calibrating the program, then draw a box at that location in the live preview so they know to keep their face within.

### CODE REVIEW

File: IrisSoftware/main.py [hunk 1]
```diff
@@ -22,9 +22,11 @@ def __init__(self) -> None:
             self.shouldExit = False
             self.isCalibrated = False
             self.calibrationEyeCoords: list[list[tuple]] = []
+            self.faceBoxes = []
+            self.faceBox = None
             self.lastCursorPos = pyautogui.position()
             self.skipMouseMovement = False
-            self.interpolatorType = InterpolationType.LINEAR_REGRESSION #InterpolationType.JOYSTICK
+            self.interpolatorType = InterpolationType.TANGENT_LINEAR_REGRESSION # InterpolationType.TANGENT_LINEAR_REGRESSION
```
Comment (zacowan) [inline IrisSoftware/main.py]: Do we still need this comment here, or can we get rid of it?

**Review by zacowan — COMMENTED** [review]
Code looks good and works as intended, the face bounding box looks great. Inside of `main.py`, could you run our auto-formatter and push it with the formatted code? When testing locally I noticed there were a few areas that the auto-formatter adjusted when I commented out the mouse movement.

File: IrisSoftware/main.py (hunk 1 above)
Comment (Scoobydoo181) [reply IrisSoftware/main.py]: We can get rid of it

**Comment by Scoobydoo181** [comment]
I actually don't have any auto-formatters installed. I saw from last time the formatter  inserted a bunch of really awkward spacing in the `moveMouse()` function, so I fixed it here.

**Review by zacowan — APPROVED** [review]

**Pull request merged**